packages/wallet-connector/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    "@babel/preset-typescript",
+    "@babel/preset-env"
+  ],
+  "plugins": [
+    "@babel/transform-runtime",
+    "@babel/plugin-transform-typescript",
+    "@babel/proposal-class-properties",
+    "@babel/proposal-object-rest-spread",
+    "@babel/plugin-proposal-optional-chaining"
+  ]
+}
```

Comment:
Is it necessary to use babel in this project?

Reply:
Yep, for compile

Reply:
Where do you use babel? 
@rarible/sdk-wallet-connector package do build without babel